prow/entrypoint/run.go

```diff
@@ -122,7 +122,8 @@ func (o Options) ExecuteProcess() (int, error) {
 			case <-ctx.Done():
 			}
 		}()
-		code, err := wrapper.WaitForMarker(ctx, o.PreviousMarker)
+		prevMarkerResult := wrapper.WaitForMarkers(ctx, o.PreviousMarker)[o.PreviousMarker]
```

Comment:
Is there a compelling reason to add the new local variable? I don't see `prevMarkerResult` being used elsewhere, so it seems functionally (but possibly not semantically) equivalent.

Reply:
I used prevMarkerResult just because I changed `WaitForMarker()` to `WaitForMarkers()` which has different return values.  It has the same behavior, but slightly less clean because of the `WaitForMarkers()`'s more complicated use case.